k8s/demo/dbench/sc-cstor-single-replica.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-cstor-r1
+  annotations:
+    openebs.io/cas-type: cstor
+    cas.openebs.io/config: |
+      - name: StoragePoolClaim
+        value: "sparse-claim-auto"
+      - name: ReplicaCount
+        value: "1"
+      - name: TargetNodeSelector
+        value: |-
+            "kubernetes.io/hostname": "gke-kmova-helm-default-pool-a169cf59-kxfm"
```

Comment:
May be we can add some comments here to change the hostname or put a info in README with details ?

Reply:
I will incorporate this change in the next PR.. as this is already merged. Thank you!